src/crypto/index.js

```diff
@@ -587,6 +590,17 @@ Crypto.prototype.getEventSenderDeviceInfo = function(event) {
     return device;
 };
 
+/**
+ * Get the current end-to-end encryption config for a room
+ *
+ * @param {string} roomId The room ID to query
+ *
+ * @return {object} The current end-to-end encyption status, or null if
```

Comment:
I don't really understand why you've made this `async`, but given you have, the return type is a promise

Reply:
ah ok. I think the reason it's async was to match the rest of the crypto stuff which assumes it's going to be indexeddb backed. I'd also assumed that the Promise return type was implicit on an async function.